File: ant/src/com/appsecinc/ant/Folders.java
package com.appsecinc.ant;

import java.util.ArrayList;

public class Folders extends ArrayList<Folder> {

	private static final long serialVersionUID = 1L;

	public Folders() {
    	
    }

    public void addConfiguredFolder(Folder folder) {
		add(folder);
	}
}

File: msbuild/src/AppSecInc.LicensesCollector/Folders.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace AppSecInc.LicensesCollector
{
    class Folders : List<Folder>
    {
        public Folders()
        {

        }

        public void AddConfiguredFolder(Folder folder)
        {
            Add(folder);
        }
    }
}
